docs/specs/git.yml

```diff
@@ -23,6 +23,7 @@ outputs:
           "id": "19990166"
         }
       ],
+      "updatedAt": "2018-06-28T10:00:00",
```

Comment:
please add more test cases to cover `updatedAt` and `author`, a lot of code branches are not covered.